docs/src/main/sphinx/conf.py

```diff
@@ -139,7 +139,7 @@ def setup(app):
 }
 
 html_theme_options = {
-    'base_url': '/',
+    'base_url': 'https://trino.io/docs/current/',
```

Comment:
What else is this parameter used for?

Reply:
It does not seem to get into any other output besides the sitemap.xml file